src/frame/auction/utils.js

```diff
@@ -34,12 +41,15 @@ export const getEligible = (groups, eligibility) => {
  */
 export const getBids = async (bidders, conf, debug) => Promise.all(
 	bidders.map(async bidder => {
+		debug && echo.groupCollapsed(`auction utils: getBids => ${bidder._key}`);
 		const time0 = performance.now();
-		const { generate_bid } = await import(bidder.bidding_logic_url);
+		const { generateBid } = await import(bidder.bidding_logic_url);
```

Comment:
didn't catch this earlier.

I'd like to keep our external API aligned with the documented proposal (not with implementation deep in chromes commit logs). This will be an easy change when its made public, but it will be confusing to those who are building to the spec as written. 

If you want to pull this change into a separate PR we can discuss more there as it doesn't need to hold up the other good work done here.

Same applies to `score_ad/scoreAd` change below.

Reply:
I can see that, but the proposal is just that and not meant to be an exact copy of the implementation details.  Kleber himself has said that in the meetings. It's in their codebase (https://source.chromium.org/chromium/chromium/src/+/master:content/test/data/interest_group/bidding_logic.js) so I think it's safe to commit to ours.  Why not stay as close to what's going to be public and not force consumers to change when it does?  That seems like we'd be asking the consumers to change when it goes public, where there are issues we can point to when someone questions it or can update the docs.

Seee: https://github.com/google/fledge-shim/issues/35 and https://github.com/WICG/turtledove/issues/156

Reply:
I hope they do switch to camelCase but the referenced materials are:
1. some test files for the generateBid and scoreAd functions which are not conclusive of much
2. the google ads team (who does't seem to have any special insight into what chrome is doing) reacting to those files
3. an issue in GitHub that the Chrome team has not responded to

I think it's premature to deviate from what is still the only documentation the chrome team has given us regarding fledge. As I mentioned, it will add confusion for someone who implements to the proposal and finds their functions cannot be called because we are expecting camelCase.

I believe we should pull the camelCase updates out into a separate PR and ask Kleber at next weeks Fledge call to get clarity and perhaps ask him to update the proposal.

Reply:
Supported both models in the end for a smoother transition period.